### ISSUE TITLE
[1.16] Core Sample Drill causing reproduceable crash in certain chunks

### ISSUE THREAD

**Issue by StormCircuit** [BluSunrize/ImmersiveEngineering#4799]
### Description of the issue:
When using the core sample drill in particular chunks (no pattern that I can see, only one 'region'/area/collection of chunks adjacent to each other). When using "/ie mineral get" an error is returned. When using "/ie mineral put <any mineral>" it returns that it successfully applied the mineral but does not fix the crash and using the ie 'mineral get' command still returns an error.

### Crashlog:
https://gist.github.com/StormCircuit/15a1b95763c1ecccaade6a8ca38fc974

### Versions & Modlist
Forge 36.1.4. 
IE 4.2.4-134
(world was generated with other mods, crash occurs with just IE)

**Comment by StormCircuit** [comment]
let me know if you'd like the world file.

**Comment by StormCircuit** [comment]
So I setup a test environment to narrow it down: with the IE world data deleted the crashes in the previously mentioned regions stop. I tried pulling the IE world data from the original base pregen'd world and it still crashed. So the corruption or otherwise cause in the files occurred when I first created and pre-gen'd the world.

When taking the world data from a separate, presumably healthy, world of the same pregen size (4500 in properties file) the crashes disappear and minerals can be found on the previously affected regions.

Here is the old, crashing, save data:
[ImmersiveEngineering-SaveData.zip](https://github.com/BluSunrize/ImmersiveEngineering/files/6402695/ImmersiveEngineering-SaveData.zip)
